<launch>

	<arg name="robot_name" />

	<!-- Run AMCL -->
	<include file = "$(find warehousetest)/config/navigation/amcl.launch.xml">
		<arg name="robot_name" value="$(arg robot_name)"/>
	</include>	










</launch>